CONTRIBUTING.md

```diff
@@ -1,194 +1,194 @@
-
-- You don’t get a response. As we are all trying to balance our work-life ratio, please give us some more time to catch up with your request.
-
-Committers/reviewers will be added to a CONTRIBUTORS.txt file.
-
-## Communication and Tools
-### Roles
-This open source project contains the following types of people:
-
-- Author: The person/s or organization that created the project. The association itself.
-- Owner: The person/s who has administrative ownership over the organization or repository. The association itself.
-- Maintainers: Contributors who are responsible for driving the vision, managing the organizational aspects of the project and handle contributions. (They may also be authors or owners of the project.) The Project Engineering Committee, described in the Governance Charter will execute this tasks with defined sub roles:
-  - Integration Manager
-Is responsible for handling and validating pull requests regarding the contribution guidelines and the overall strategic scope.
-  - (Dictator, if any) 
-
-- Contributors: Everyone who has contributed something back to the project.
-- Community Members: People who use the project. They might be active in conversations or express their opinion on the project’s direction.
-
-### Communication
-
-The best way to get in touch with us is directly via GitHub. 
-
-Also feel free to write us an email: hello@aposin.org.
-
-If you have any questions please contact us directly via GitHub. 
-
-- Issue tracker: Where people discuss issues related to the project.
-For issue tracking and communication at this OSS project the build in GitHub issue section is the center of communication for all bugs, features and discussions.
-- Pull requests: Where people discuss and review changes that are in progress.
-- Discussion forums or mailing lists: Some projects may use these channels for conversational topics (for example, “How do I…“ or “What do you think about…“ instead of bug reports or feature requests). Others use the issue tracker for all conversations. For now there is no specific forum set up.
-
-### Tools
-Currently there are no specific third party tools for communication or issue tracking in use.
-
-## General GitHub resources on how to start with contributions
-
-- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
-- [Using Pull Requests](https://help.github.com/articles/about-pull-requests/)
-- [GitHub Help](https://help.github.com)
-
-
+
+
+
+# Contributing to LicenseScout
+Thank you for considering a contribution to LicenseScout! 
+
+We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great. 
+
+This guide explains how to:
+
+* maximize the chance of your changes being accepted
+* work on the this project's code base
+* get help if you encounter trouble
+
+## Prerequisite
+
+#### Code of Conduct
+
+[fork]: https://help.github.com/en/github/getting-started-with-github/fork-a-repo

```

Comment:
Are just those two links changed or there is a reason why the whole file is changed? If it is because of the automatic line break, the problem is already fixed but you need to clone again the repository. Could you please double check it? Thanks!